test/Microsoft.AspNet.Hosting.Tests/HostingEngineTests.cs

```diff
@@ -26,9 +27,8 @@
 
 namespace Microsoft.AspNet.Hosting
 {
-    public class HostingEngineTests : IServerFactory
+    public class HostingEngineTests
```

Comment:
To reduce the test churn, make this type implement both IServer and IServerFactory